src/commands/touch.ts

```diff
@@ -0,0 +1,35 @@
+import { CommandContext } from "../command_handler.ts";
+import { resultFromCode } from "../result.ts";
+import { parseArgKinds } from "./args.ts";
+
+export async function touchCommand(context: CommandContext) {
+  try {
+    await executetouch(context.args);
+    return resultFromCode(0);
+  } catch (err) {
+    context.stderr.writeLine(`touch: ${err?.message ?? err}`);
+    return resultFromCode(1);
+  }
+}
+
+async function executetouch(args: string[]) {
+  const flags = parseArgs(args);
+  for (const path of flags.paths) {
+    const f = await Deno.create(path);
+    f.close();
+  }
+}
+
+interface touchFlags {
+  paths: string[];
+}
+
+export function parseArgs(args: string[]): touchFlags {
+  const paths = [];
+  for (const arg of parseArgKinds(args)) {
+    if (arg.kind === "Arg") paths.push(arg.arg);
```

Comment:
```
$ touch --test
touch: unrecognized option '--test'
```

We should probably also error for unrecognized options. Also, this code should ensure all `arg.kind` types are handled such that if a new `arg.kind` type were added we would get a type error here.

Reply:
Yup, should be good. Thanks!